OWML.Common/IModInteraction.cs

```diff
@@ -8,7 +8,7 @@ public interface IModInteraction
         IList<IModBehaviour> GetDependants(string dependencyUniqueName);
         IList<IModBehaviour> GetDependencies(string uniqueName);
         IModBehaviour GetMod(string uniqueName);
-        T GetMod<T>(string uniqueName) where T : IModBehaviour;
+        T GetApi<T>(string uniqueName) where T : class;
```

Comment:
Sure about calling it Api instead of Mod?

Reply:
Well it does return the API of the mod... so maybe GetModAPI<T>() ?

Reply:
Good idea :)